### ISSUE TITLE
Parsing quote-printable body content problem

### ISSUE THREAD

**Issue by mrsoto** [jhermsmeier/node-envelope#13]
Hello.

I have the following body content which has line feeds but the envelope part corresponding to the text/plain body part came without line feeds. 

```
------=_Part_1796134_734941703.1461597339659
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

=C2=A0
Get Firefox!

     On Thursday, April 21, 2016 8:19 PM, zzzz zzz <zzzzz@yahoo.com> wr=
ote:
=20

 =C2=A0
Get Firefox!
```

I need to display this body content in an webmail like page but the space between "Get Firefox! and On..." is lost

As far I understand this happened in:

```
Envelope@envelope.js:22
Body@index.js:52
MME.decodeQP@mime.js:148
```

**Comment by jhermsmeier** [comment]
Hey, thanks for reporting this!
I just ran a quick test with what you posted, and this is what I get:
```js
{
  '0': {
    '0': ' Get Firefox!     On Thursday, April 21, 2016 8:19 PM, zzzz zzz <zzzzz@yahoo.com> wrote:   Get Firefox!',
    header: {
      contentType: {
        mime: 'text/plain',
        charset: 'UTF-8'
      },
      contentTransferEncoding: 'quoted-printable'
    }
  },
  header: {
    contentType: {
      mime: 'multipart/alternative',
      boundary: '----=_Part_1796134_734941703.1461597339659'
    }
  }
}
```
So the space seems to be there, just not the newlines – which I guess is what you mean?

**Comment by mrsoto** [comment]
Yes \n and or \r, they are lost.

This is an email send (part) from mail.yahoo.com

I did not see any way to get raw content

**Comment by jhermsmeier** [comment]
Should be fixed in `envelope@1.0.4` now – let me know if you run into any further issues!
